Add a JavaScript quickstart (#1390)
ngrok/ngrok-docs: docs/getting-started/javascript.mdx

---
sidebar_label: JavaScript
title: ngrok JavaScript SDK Quickstart
pagination_next: universal-gateway/overview
---

import TabItem from "@theme/TabItem";
import Tabs from "@theme/Tabs";
import ConfigExample from "/src/components/ConfigExample.tsx";
import SecureYourApp from "./_secure-your-app.mdx"

[The ngrok JavaScript (Node.js) SDK](https://ngrok.github.io/ngrok-javascript/) enables you to quickly and efficiently serve Node.js applications on the internet without the need to configure low-level network primitives like IPs, certificates, load balancers, or ports.

> This quickstart uses ngrok's JavaScript SDK to put a Node.js app running on your local device online and secure it so anyone with a Google account can access it.

Reviewer: ```suggestion
This quickstart uses ngrok's JavaScript SDK to put a Node.js app running on your local device online and secure it by requiring visitors to log in with a Google account to access it.
```
Feels a bit off to say "secures it," but then frame it like "anyone with a Google account can access it".

Author: good call
